src/components/apply/avatar-radio.tsx

```diff
@@ -0,0 +1,58 @@
+import { ChangeEvent, useState } from "react";
+import Avatar from "./avatar";
+
+type AvatarType =
+  | "Wildlife Wanderer"
+  | "City Cruiser"
+  | "Foodie Fanatic"
+  | "Beach Bum"
+  | null
+  | undefined;
+
+interface AvatarRadioProps {
+  value: AvatarType;
+  onChange: (avatar: AvatarType) => void;
+}
+
+const avatarOptions: AvatarType[] = [
+  "Wildlife Wanderer",
+  "City Cruiser",
+  "Foodie Fanatic",
+  "Beach Bum",
+];
+
+const AvatarRadioGroup = ({ value, onChange }: AvatarRadioProps) => {
+  const [selectedAvatar, setSelectedAvatar] = useState<AvatarType | null>(
+    value,
+  );
+
+  const onSelect = (e: ChangeEvent<HTMLInputElement>) => {
+    const avatar = e.target.value as AvatarType;
+    setSelectedAvatar(avatar);
+    onChange(avatar);
+  };
+
+  return (
+    <div className="grid grid-cols-2 gap-8">
+      {avatarOptions.map((option, index) => (
+        <label key={index} className="flex w-full cursor-pointer">
+          <input
+            type="radio"
+            name="avatar"
+            checked={selectedAvatar === option}
+            onChange={onSelect}
+            className="hidden"
+          />
+          <Avatar
```

Comment:
I didn't know where to put this, but we need different styling for selected and deselected inputs.

The design:
<img width="623" alt="image" src="https://github.com/user-attachments/assets/f338b9d6-863b-43d8-997b-a2c0ca394c67">

Current:
<img width="406" alt="image" src="https://github.com/user-attachments/assets/66efc213-b53e-4658-95b5-e2e7dec97365">